.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,68 @@
+name: PHP Static Analysis
```

Comment:
We should still be able to do this by generating a baseline on master branch automatically. I actually managed to get that part working in my PR.